packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -37,6 +37,12 @@ describe('ui:create:atomic', () => {
       projectName
     );
 
+    await buildTerminal
```

Comment:
Probably something wrong i'm doing here... even if I can make it work locally

Reply:
What's the problem doc?